routes/object.js

```diff
@@ -51,3 +30,24 @@ module.exports = (elastic, config) => ({
     }
   }
 });
+
+function HTMLResponse (request, reply, elastic, config) {
```

Comment:
I prefer to create a new file for this function (and maybe we can reuse it on other endpoint). It allow us also to create a specific unit test for it. But it's just my preference and it's also fine for me the way it is.